modules/hcp-virt-taints-tolerations.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-virt-taints-tolerations_{context}"]
+= Custom taints and tolerations
+
+For {hcp} on {VirtProductName}, by default, pods for a hosted cluster tolerate the `control-plane` and `cluster` taints. However, you can also use custom taints on nodes so that hosted clusters can tolerate those taints on a per-hosted-cluster basis by setting `HostedCluster.spec.tolerations`.
+
+:FeatureName: Passing tolerations for a hosted cluster
+include::snippets/technology-preview.adoc[]
+
+.Example configuration
+[source,yaml]
+----
+  spec:
+    tolerations:
+    - effect: NoSchedule
+      key: kubernetes.io/custom
+      operator: Exists
+----
+
+You can also set tolerations on the hosted cluster while you create a cluster by using the `--tolerations` hcp CLI argument.
+
+.Example CLI argument
+[source,terminal]
+----
+--toleration="key=kubernetes.io/custom,operator=Exists,effect=NoSchedule"
+----
+
+For fine granular control of hosted cluster pod placement on a per-hosted-cluster basis, use custom tolerations with `nodeSelectors`. You can co-locate groups of hosted clusters and isolate them from other hosted clusters. You can also place hosted clusters in infra and master nodes.
```

Comment:
Just wanted to check - is "master" here totally necessary for technical accuracy, or can some other term be used? I think we're supposed to avoid that term whenever possible.

Reply:
Good catch! I think that can be changed to "control plane".